src/OXRS_WT32.cpp

```diff
@@ -167,12 +171,12 @@ void _getConfigSchemaJson(JsonVariant json)
     _mergeJson(properties, _fwConfigSchema.as<JsonVariant>());
   }
 
-  // SHT20 sensor config
-  if (_climateSensorFound)
+  // sensor config only if ESP is S3 or SHT20 found
+  if (strcmp(ESP.getChipModel(), "ESP32-S3") == 0 || _sht20Found)
```

Comment:
Sorry Moin, I am just a bit confused why you have this check here, but everywhere else you are using `#if defined(CONFIG_IDF_TARGET_ESP32S3)`? I am sure they have the same result, but it makes the code a little easier to read if they are consistent?

Reply:
The internal temp sensor is ESP-model dependent. The corresponding library (temp_sensor.h) is only available when compiling for ESP32S3. Therefor everything that deals with the internal temp sensor has to be wraped with compile time flags (#if def ...).
The SHT20 can be attached to all panels and it has to be compiled into each version of the FW. The existence of the SHT is checked at runtime and therefore the flag _sht20Found is used.
The Sensor Config itself is HW independent but the decission to present it depends on the compile time ESP detection and the run time SHT20 detection. ESP.getChipModel() helps here to combine the two. and present the config for all ESP models.